docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       ES_HOST: es
       ES_INDEX: profiles
       CHANGE_THRESHOLD: '0.96'
+      ES_TIMEOUT_SECONDS: '30'
```

Comment:
I'm wondering why the timeout is only 30 for the local app? Also the default is 30 so is this needed?